### PR TITLE
 Praos: use larger nonce stabilization window starting in Conway 

### LINKED ISSUE #297: Switch Epoch structure from 4-3-3 to 3-3-4
This is important for Genesis work. Ideally it should be done before Conway, however if it can be implemented in isolation within a single rule, then an intra-era hardfork could be possible as well.

Timeline for this is sometime next year (unclear when exactly).

Any questions about this should be directed to Christian Badertscher and Duncan

### PR DESCRIPTION
**Pull request by amesgen**
Closes #297, closes https://github.com/IntersectMBO/cardano-ledger/issues/2478, closes https://github.com/IntersectMBO/cardano-ledger/issues/1914

See erratum 17.3 (amended in https://github.com/IntersectMBO/cardano-ledger/pull/4015) in the Shelley ledger specs for context.

### CODE REVIEW

**Comment by amesgen** [comment]
Note: We should also inform other teams about this change; in particular such that `cardano-cli` commands reporting the leader schedule can be adapted (as they have to implement the logic themselves until something like #390) lands.

**Review by jasagredo — APPROVED** [review]
Looks good. The glossary also is understandable.

**Comment by amesgen** [comment]
We should not merge this for the Consensus release that goes into node 8.9 as that release it supposed to be rather minimal (and this PR requires all existing Conway chains to be truncated, in particular Sanchonet).

File: ouroboros-consensus-protocol/changelog.d/20240206_174209_alexander.esgen_change_epoch_structure.md
Comment (nfrisby) [inline ouroboros-consensus-protocol/changelog.d/20240206_174209_alexander.esgen_change_epoch_structure.md:4]: where -> when.... or even better, "in which slot in the era"

File: ouroboros-consensus-cardano/src/ouroboros-consensus-cardano/Ouroboros/Consensus/Cardano/Node.hs
Comment (nfrisby) [inline ouroboros-consensus-cardano/src/ouroboros-consensus-cardano/Ouroboros/Consensus/Cardano/Node.hs:735]: Maybe add a comment here, at least saying `see 'partialConsensusConfigBabbage'` or something like that

**Review by nfrisby — APPROVED** [review]
LGTM

**Comment by amesgen** [comment]
Merging now as the Consensus packages for Node 8.9 were released in #969. (Context: https://github.com/IntersectMBO/ouroboros-consensus/pull/927#issuecomment-1930377325)

**Pull request merged**
